src/test/java/com/blogify/blogapi/integration/conf/TestUtils.java

```diff
@@ -27,6 +27,8 @@ public class TestUtils {
 
   public static final FirebaseUser firebaseUserClient2 =
       new FirebaseUser("test+herilala@hei.school", "client2_firebase_id");
+  public static final FirebaseUser firebaseUserClient3 =
```

Comment:
no no, we not need to add this, it's the clent we will rune request in "     ApiClient client1Client = anApiClient(CLIENT1_TOKEN);  "
so we will never use le client 3 hear